resources/automagic_dashboards/field/State.yaml

```diff
@@ -1,6 +1,6 @@
-title: A look at your [[this]] per state
-transient_title: Here's a closer look at your [[this]] per state
-description: How [[this]] is distributed, and how different numbers are represented per state.
+title: A look at your [[this]]
+transient_title: Here's an overview of your [[this]]
+description: How many [[GenericTable]] there are per state, and how each state is represented across other categories.
```

Comment:
Are descriptions of State and Country intentionally slightly different?

Reply:
I don't want every description to feel too similar (makes it kind of annoying to scan), but I think the country one could be better.

Reply:
Never thought of that. Excellent point.